ui/src/app/data-explorer/components/designer-panel/data-settings/data-explorer-widget-data-settings.component.ts

```diff
@@ -181,7 +181,7 @@ export class DataExplorerWidgetDataSettingsComponent implements OnInit {
         this.triggerDataRefresh();
     }
 
-    addDataSource(
+    addDataStream(
```

Comment:
In the data explorer, I would stick to `Data Source` since we are working on persisted data. Using data stream can be confusing for users who would then expect real-time data.

Reply:
Yep, this is reasonable 👍🏼

Reply:
Done